package.json

```diff
@@ -42,14 +42,14 @@
   },
   "license": "LGPL-3.0",
   "scripts": {
+    "prebuild": "npm run lint",
     "build": "npm run build:main && npm run build:standalone",
     "build:main": "browserify -e ./src/main.js -t deamdify | derequire - > index.js",
     "build:standalone": "browserify -d -e ./src/main.js -t deamdify -s peaks | exorcist peaks.js.map | derequire - > peaks.js",
     "doc": "jsdoc --private --destination docs --recurse src",
     "lint": "eslint src/**/*.js test/**/*.js",
     "pretest": "npm run build",
     "test": "./node_modules/karma/bin/karma start",
-    "posttest": "npm run lint",
```

Comment:
IMHO you want to run the lint as early as possible, with the intent of exposing errors in the code.